Formula/libint.rb

```diff
@@ -0,0 +1,20 @@
+class Libint < Formula
+  desc "Library for computing electron repulsion integrals efficiently"
+  homepage "https://github.com/evaleev/libint"
+  url "https://github.com/evaleev/libint/archive/v2.7.2.tar.gz"
+  sha256 "fd0466ce9eb6786b8c5bbe3d510e387ed44b198a163264dfd7e60b337e295fd9"
+  license "GPL-2.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "boost"
```

Comment:
it should also depends on "eigen"?
```suggestion
  depends_on "boost"
  depends_on "eigen"
```


```
/opt/homebrew/Cellar/libint/2.7.2/include/libint2/./engine.impl.h:31:10: fatal error: 'Eigen/Core' file not found
#include <Eigen/Core>
         ^~~~~~~~~~~~
1 error generated.
```